src/main/resources/com/sonyericsson/hudson/plugins/gerrit/trigger/GerritManagement/index.jelly

```diff
@@ -1,3 +1,4 @@
+<?jelly escape-by-default='true'?>
```

Comment:
Remember that this change can cause regressions. If the original file had

```xml
${it.someUntaintedDataWhichMightContainHTML}
```

you need to switch it to

```xml
<j:out value="${it.someUntaintedDataWhichMightContainHTML}"/>
```